### PR TITLE
Add Statemine USDT 

### PR DESCRIPTION
**Pull request by josietyleung**

### CODE REVIEW

File: _non-cosmos/statemine/assetlist.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      ],
+      "base": "130",
+      "name": "Statemine",
+      "display": "usdt",
+      "symbol": "USDT",
+      "traces": [
+        {
+          "type": "bridge",
+          "counterparty": {
+            "chain_name": "statemine",
```
Comment (JeremyParish69) [inline _non-cosmos/statemine/assetlist.json:24]: the bridge trace from itself (from statemine to statemine) doesn't make sense....

File: _non-cosmos/picasso/assetlist.json [hunk 2]
```diff
@@ -77,6 +77,32 @@
...
+      "display": "usdt",
+      "symbol": "USDT",
+      "traces": [
+        {
+          "type": "bridge",
+          "counterparty": {
+            "chain_name": "statemine",
+            "base_denom": "130"
+          },
+          "provider": "Statemine"
```
Comment (JeremyParish69) [inline _non-cosmos/picasso/assetlist.json:103]: I think we've been using something like Kusama Parachain or Kusama Relay for these transfers

**Review by bitcoinfinancier — APPROVED** [review]

File: _non-cosmos/statemine/assetlist.json (hunk 1 above)
Comment (josietyleung) [reply _non-cosmos/statemine/assetlist.json]: ah updated

File: _non-cosmos/picasso/assetlist.json (hunk 2 above)
Comment (josietyleung) [reply _non-cosmos/picasso/assetlist.json:103]: that was for KSM which came from the Kusama relay chain. Kusama relay chain is connected to Picasso. 
Statemine is also directly connected to Picasso so USDT will transfer directly from Statemine > Picasso

**Pull request merged**